make/git/git_pullall.sh

```diff
@@ -44,6 +44,8 @@ if test -z "${has_remote}"; then
 else
```

Comment:
The submodule update should be in the if case above too

Reply:
Done.